Part2_Mapplots/ex2_4_colorbars.ipynb

```diff
@@ -1,20 +1,5 @@
 {
```

Comment:
<p>Check space (difficult to see in reviewnb)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/C2SM/pyvis/pull/95/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>